desafio-03/KarineFBrito/java/Palindromo.java

```diff
@@ -0,0 +1,35 @@
+import java.util.Scanner;
+
+public class Palindromo {
+  public static void main(String[] args) {
+    Scanner sc = new Scanner(System.in);
+    System.out.println("Escolha o número inicial: ");
+    int n1 = sc.nextInt();
+    System.out.println("Escolha o numero final: ");
+    int n2 = sc.nextInt();
+    if (n1 < 0 || n2 < 0) {
+      System.out.println("Os números não podem ser negativos!");
+      sc.close();
+      return;
```

Comment:
Teu programa está quase completo. Mas sempre que abortar um programa, executando no terminal, adicione o status para indicar se foi sucesso ou erro: https://codegym.cc/groups/posts/systemexit-in-java

Dê uma lida neste artigo que mandei, vão ser duas alterações simples que tem que adicionar em teu programa quando o input do usuário for incorreto